[gdal] build failure

**Host Environment**
- OS: Windows Server 2019
- Compiler: Visual Studio Enterprise 2019

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install gdal[core,tools]`

**Failure logs** 
```
 CMake Error at scripts/cmake/vcpkg_copy_tools.cmake:70 (message):
  Couldn't find tool "testepsg":

      "C:/vcpkg/packages/gdal_x64-windows/bin/testepsg.exe" does not exist
Call Stack (most recent call first):
  ports/gdal/portfile.cmake:172 (vcpkg_copy_tools)
  scripts/ports.cmake:142 (include)


Error: Building package gdal:x64-windows failed with: BUILD_FAILED
```
**Additional context**

The [gdal portfile](https://github.com/microsoft/vcpkg/blob/master/ports/gdal/portfile.cmake) includes `testepsg`, but that was removed from the default build for gdal in https://github.com/OSGeo/gdal/commit/0272378cc7b16975e3a85585e175ec8b8fffcaa1. I expect that needs to be removed from the portfile.

--- a/ports/gdal/portfile.cmake
+++ b/ports/gdal/portfile.cmake
@@ -166,7 +166,6 @@ if (VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
             ogrinfo
             ogrlineref
             ogrtindex
-            testepsg
             )
         # vcpkg_copy_tools removed the bin directories for us so no need to remove again
         vcpkg_copy_tools(TOOL_NAMES ${GDAL_EXES} AUTO_CLEAN)

--- a/ports/gdal/vcpkg.json
+++ b/ports/gdal/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gdal",
   "version-semver": "3.4.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The Geographic Data Abstraction Library for reading and writing geospatial raster and vector data",
   "homepage": "https://gdal.org",
   "supports": "!(arm & windows)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2362,7 +2362,7 @@
     },
     "gdal": {
       "baseline": "3.4.0",
-      "port-version": 1
+      "port-version": 2
     },
     "gdcm": {
       "baseline": "3.0.7",

--- a/versions/g-/gdal.json
+++ b/versions/g-/gdal.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f0f7e3cd7a86246d31f892678b2e06d01c455c5b",
+      "version-semver": "3.4.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "d5580a73ced0a55c4659874d1755239c76e9168c",
       "version-semver": "3.4.0",
       "port-version": 1

